index.html

```diff
@@ -2,241 +2,277 @@
-              <img class="s-m-icon" src="Image/github.svg" alt="github">
-            </a>
-          </li>
-          <li class="social-media-icon">
-            <a href="#social-media-icon" target="_blank" rel="noopener noreferrer">
-              <img class="s-m-icon" src="Image/angelist.svg" alt="Angelist">
-            </a>
-          </li>
-        </ul>
-        <button class="about__myself__btn btn" type="button">Get my resume</button>
-        <ul class="skill__section">
-          <li class="skill__section__container">
-            <div class="skill__info">
-              <h2 class="skill__title">Languages</h2>
-              <img src="Image/arrowdown.svg" alt="arrow-dropdowm">
-            </div>
-            <ul class="language__list">
-              <li class="language">
-                <img src="Image/JS.svg" alt="javascript">
-                <h2 class="language__name">JavaScript</h2>
-              </li>
-              <li class="language">
-                <img src="Image/HTML.svg" alt="Html">
-                <h2 class="language__name">HTML</h2>
-              </li>
-              <li class="language">
-                <img src="Image/CSS.svg" alt="Css">
-                <h2 class="language__name">CSS</h2>
-              </li>
-            </ul>
-          </li>
-          <li class="skill__info">
-            <h2 class="skill__title">Framework</h2>
-            <img src="Image/arrowright.svg" alt="arrow-dropdowm">
-          </li>
-          <li class="line"></li>
-          <li class="skill__info">
-            <h2 class="skill__title">Skills</h2>
-            <img src="Image/arrowright.svg" alt="arrow-dropdowm">
-          </li>
-          <li class="line"></li>
-        </ul>
-      </section>
-    
+        </li>
+        <li class="skill__info">
+          <h2 class="skill__title">Framework</h2>
+          <img src="Image/arrowright.svg" alt="arrow-dropdowm">
+        </li>
+        <li class="line"></li>
+        <li class="skill__info">
+          <h2 class="skill__title">Skills</h2>
+          <img src="Image/arrowright.svg" alt="arrow-dropdowm">
+        </li>
+        <li class="line"></li>
+      </ul>
+    </section>
+  </article>
+
+  <!-- Footer -->
+  <footer class="footer">
+    <h2 class="footer-title">Contact me</h2>
+    <p class="footer-text">If you have an application you are interested in developing, a feature that you need built or
+      a project that needs coding. I’d love to help with it</p>
+    <form class="form" action="https://formspree.io/f/xbjqvdnv" method="post">
+      <ul class="formul">
+        <li class="formul1">
+          <label for="fullName">
+            <input type="text" name="name" id="fullName" placeholder="Yeremias" maxlength="30" required>
+          </label>
+        </li>
+        <li class="formul2">
+          <label for="email">
+            <input type="email" name="email" id="email" placeholder="natajayanj14@gmail.com" required>
```

Comment:
- [ ] [OPTIONAL] I know that the Figma design shows the name "Yeremias" and the email placeholder "natajayan14@gmail.com", but I think it could be a good idea to use placeholders like "Your name" and "Your email" for a better user experience :wink: